Cubical/HITs/Nullification/Properties.agda

```diff
@@ -83,6 +112,26 @@ module _ {ℓα ℓs ℓ ℓ'} {A : Type ℓα} {S : A → Type ℓs} {X : Type
                      (λ i → elim nY g (p s i))
           q₂ j i = toPathP⁻ (λ j → Y (≡spoke p s j i)) (λ j → q₁ j i) j
 
+NullRecIsPathSplitEquiv : ∀ {ℓα ℓs ℓ ℓ'} {A : Type ℓα} {S : A → Type ℓs} {X : Type ℓ} {Y : Type ℓ'} → (isNull S Y) →
+                          isPathSplitEquiv {A = (Null S X) → Y} (λ f → f ∘ ∣_∣)
+sec (NullRecIsPathSplitEquiv nY) = rec nY , λ _ → refl
+secCong (NullRecIsPathSplitEquiv nY) f f' = (λ p → funExt (elim (λ _ → isNull≡ nY) (funExt⁻ p))) , λ _ → refl
+
+NullRecIsEquiv : ∀ {ℓα ℓs ℓ ℓ'} {A : Type ℓα} {S : A → Type ℓs} {X : Type ℓ} {Y : Type ℓ'} →  (isNull S Y) →
+                          isEquiv {A = (Null S X) → Y} (λ f → f ∘ ∣_∣)
+NullRecIsEquiv nY = toIsEquiv _ (NullRecIsPathSplitEquiv nY)
+
+NullRecEquiv : ∀ {ℓα ℓs ℓ ℓ'} {A : Type ℓα} {S : A → Type ℓs} {X : Type ℓ} {Y : Type ℓ'} → (isNull S Y) →
+                          ((Null S X) → Y) ≃ (X → Y)
+NullRecEquiv nY = (λ f → f ∘ ∣_∣) , (NullRecIsEquiv nY)
+
+
+NullPreservesProp : ∀ {ℓα ℓs ℓ} {A : Type ℓα} {S : A → Type ℓs} {X : Type ℓ} →
+                    (isProp X) → isProp (Null S X)
+
+NullPreservesProp {S = S} pX u = elim (λ v' → isNull≡ (isNull-Null S))
+  (λ y → elim (λ u' → isNull≡ (isNull-Null S) {x = u'}) (λ x → cong ∣_∣ (pX x y)) u)
+
```

Comment:
According to the naming conventions (which are a bit fuzzy on this point), these should start with lower case letters. However, ```Null``` is somewhat like a symbol here, so maybe the names should be left like they are. I tend to leaving things like they are - I just wanted to make you aware of the conflict with the convention.

Reply:
I agree to leave as is if you think it's fine. I think capitals are used this way sometimes already, e.g. `Maybe≡SumUnit`.